decidim-initiatives/config/locales/en.yml

```
@@ -146,8 +146,12 @@ en:
             time_and_date: Time and date
             timestamp: Timestamp
       titles:
+        edit_initiative_type_scope: Edit initiative type scope
         initiatives: Initiatives
+        initiatives_settings: Initiative settings
```

Consistency with the view title:
```suggestion
        initiatives_settings: Settings for initiatives
```